modules/oe_theme_content_project/templates/field--node--extra-field-oe-theme-content-project-funding--oe-project.html.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * @file
+ * Project Funding field template.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% if items.0.content['#items'] %}
+  {% set _items = items.0.content['#items'] %}
+{% else %}
+  {% set _items = [] %}
+{% endif %}
+
+<div id="project-funding">
+  {% if _items is not empty %}
+    <h2 class="ecl-u-type-heading-2">{{ label }}</h2>
+    <ul class="ecl-unordered-list ecl-unordered-list--divider">
+      {% for item in _items %}
+        <li class="ecl-unordered-list__item">
+          <div class="ecl-u-type-s ecl-u-type-color-grey-75 ecl-u-type-family-alt"><span class="ecl-u-type-uppercase">{{ item.fieldlabel }}</span></div>
+          <div class="ecl-u-d-inline-block ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-type-family-alt ecl-u-mt-xs">
+            {% include '@ecl-twig/link' with {
+              'link': {
+                'type': 'standalone',
+                'label': item.link.text,
+                'path': item.link.url
+              }
+            } only %}
+          </div>
```

Comment:
Let's use directly the list item pattern in here.

Reply:
Changed to use list item pattern